fancy-dabbrev.el

```diff
@@ -421,14 +448,14 @@ nil."
 
 (defun fancy-dabbrev--get-first-expansion ()
   "[internal] Return the first expansion candidate."
-  (fancy-dabbrev--with-suppressed-message
-    (dabbrev--reset-global-variables)
-    (setq fancy-dabbrev--entered-abbrev (dabbrev--abbrev-at-point))
-    ;; Messages from dabbrev--find-expansion ("Scanning for dabbrevs...done")
-    ;; are suppressed since they are annoying when searching for a candidate
-    ;; for the preview.
-    (dabbrev--find-expansion
-     fancy-dabbrev--entered-abbrev 0 dabbrev-case-fold-search)))
+  (dabbrev--reset-global-variables)
+  (setq fancy-dabbrev--entered-abbrev (dabbrev--abbrev-at-point))
+  ;; Messages from dabbrev--find-expansion ("Scanning for dabbrevs...done")
+  ;; are suppressed since they are annoying when searching for a candidate
+  ;; for the preview.
+  (fancy-dabbrev--without-progress-reporter
```

Comment:
Looks like "Scanning for dabbrevs...done" messages are still printed on `fancy-dabbrev-expand`. Did you consider plugging that as well?

Would something like

```diff
--- a/fancy-dabbrev.el
+++ b/fancy-dabbrev.el
@@ -406,7 +406,8 @@ nil."
                       (not fancy-dabbrev--preview-overlay-was-visible))))
         (setq fancy-dabbrev--expansions nil)
       (if (fancy-dabbrev--any-bound-and-true fancy-dabbrev-no-expansion-for)
-          (dabbrev-expand nil)
+          (fancy-dabbrev--with-suppressed-message
+            (dabbrev-expand nil))
         (add-hook 'post-command-hook #'fancy-dabbrev--post-command-hook)
         (if last-command-did-expand
             (fancy-dabbrev--expand-again t)
@@ -541,10 +542,11 @@ point."
           new-expansions)
       (while (and (< i fancy-dabbrev-menu-height)
                   (setq expansion
-                        (dabbrev--find-expansion
-                         fancy-dabbrev--entered-abbrev
-                         0
-                         dabbrev-case-fold-search)))
+                        (fancy-dabbrev--with-suppressed-message
+                          (dabbrev--find-expansion
+                           fancy-dabbrev--entered-abbrev
+                           0
+                           dabbrev-case-fold-search))))
         (setq new-expansions (cons expansion new-expansions))
         (setq i (1+ i)))
       (setq fancy-dabbrev--expansions
```

make sense?

Reply:
Done, although I used `fancy-dabbrev--without-progress-reporter` instead of `fancy-dabbrev--with-suppressed-message` which is working for me.